core-meetings/2018-08-cli-meeting.md

```diff
@@ -0,0 +1,39 @@
+# The CLI and its future: August 2018 Core Meeting
+
+As stewards of React Native engineers worldwide, we've contemplated ways to improve alignment and visibility of the framework's direction, where community members and Facebook alike joined to discuss issues and propose responses.
+
+The community team is made up of volunteers who help maintain React Native, contribute to education materials, and improve the React Native developer's environment.
+
+## The core discussion
+
+This month, the group chose to discuss React Native's CLI tooling and its future.
```

Comment:
We have multiple groups and they are supposed to work simultaneously rather than one group meeting once a month.